phaser/column.py

```diff
@@ -149,6 +149,41 @@ def fix_value(self, value):
         return value
 
 
+class BooleanColumn(Column):
+    TRUE_VALUES = ['t', 'true', '1', 'yes', 'y']
+    FALSE_VALUES = ['f', 'false', '0', 'no', 'n']
+
+    def __init__(self,
+                 name,
+                 required=True,
+                 null=False,  #Test this being either way
```

Comment:
I am starting to think that the parameter should be named something like `null_allowed`.  I can't keep straight in my head what the `null` parameter means.